po/es.po

```diff
@@ -630,48 +655,48 @@ msgstr[1] "%d nuevos atajos agregados (contexto: \"%s\")"
 
 #, fuzzy
```

Comment:
Hi,
Some translations are still marked as "fuzzy" but translated, you must remove this fuzzy marker (multiple places).
Thanks.